xarray/backends/rasterio_.py

```diff
@@ -174,11 +174,48 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
 
     You can generate 2D coordinates from the file's attributes with::
 
-        from affine import Affine
-        da = xr.open_rasterio('path_to_file.tif')
-        transform = Affine.from_gdal(*da.attrs['transform'])
-        nx, ny = da.sizes['x'], da.sizes['y']
-        x, y = np.meshgrid(np.arange(nx)+0.5, np.arange(ny)+0.5) * transform
+        >>> from affine import Affine
+        >>> import xarray as xr
+        >>> import numpy as np
```

Comment:
this is not needed, we always populate the doctest namespace with `xarray`, `numpy`, and `pandas`:
https://github.com/pydata/xarray/blob/c7c4aae1fa2bcb9417e498e7dcb4acc0792c402d/conftest.py#L26-L41

```suggestion
```